server/routes/middleware/populateCurrentUser.js

```diff
@@ -11,14 +11,18 @@ module.exports = userService => async (req, res, next) => {
         logger.info('Get user credentials from Redis cache.')
       }
       res.locals.user = { ...JSON.parse(getReply), ...res.locals.user }
-      return next()
+      if (!res.locals.user.name || !res.locals.user.userId) {
+        logger.warn('Error getting user credentials from Redis cache.', getReply)
```

Comment:
Can you log the user object as well here please? Just in case it is missing other details supposed to be populated from the JWT